packages/api/src/graphql/query.private.ts

```diff
@@ -431,6 +437,18 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
 
     // Comments
     // =======
+
+    comment: {
+      type: GraphQLComment,
+      args: {id: {type: GraphQLID}},
+      resolve(root, {id}, {authenticate, dbAdapter}) {
+        const {roles} = authenticate()
+        authorise(CanGetComment, roles)
+
+        return dbAdapter.comment.getCommentById(id)
```

Comment:
I think we should check if id does not equal null?

example from same file:

`
 if (id == null) {
   throw new UserInputError('You must provide `id`')
}
`